unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/UpdateEventHandler/UpdateEventHandler.cs

```diff
@@ -9,10 +9,11 @@ namespace DCL
     public class UpdateEventHandler : IUpdateEventHandler
     {
         private UpdateDispatcher dispatcher;
+        private const string UPDATE_DISPATCHER = "_UpdateDispatcher";
 
         public UpdateEventHandler ()
         {
-            var go = new GameObject("_UpdateDispatcher");
+            var go = new GameObject(UPDATE_DISPATCHER);
```

Comment:
We are never using the object name to find stuff.

I think this is only instantiated once, so it wont be a big deal, but we could do something like this:
```suggestion
            var go = new GameObject();
   #if UNITY_EDITOR
            go.name = UPDATE_DISPATCHER;
    #endif
```